utility/utils.py

```diff
@@ -674,10 +674,8 @@ def email_results(test_result):
         send_to_cephci (optional [bool]): send to cephci@redhat.com as well as user email
     Returns: None
     """
-    cfg = get_cephci_config().get("email")
-    if not cfg:
-        return
     sender = "cephci@redhat.com"
+    cfg = get_cephci_config().get("email", {"address": sender})
```

Comment:
This would always send an email. We should not send an email if the user has not provided the email address